frontend/src/Components/Charts/TrafficStops/TrafficStops.js

```diff
@@ -292,7 +295,7 @@ function TrafficStops(props) {
               label="Stop Purpose"
               value={purpose}
               onChange={handleStopPurposeSelect}
-              options={[PURPOSE_DEFAULT].concat(STOP_TYPES)}
+              options={[PURPOSE_DEFAULT].concat(stopTypes)}
```

Comment:
I think this may also need to be in the data table:

![Screen Shot 2023-03-07 at 5 25 51 PM](https://user-images.githubusercontent.com/129679/223567795-596f5223-c2b1-47c2-b585-68ea39538885.png)